ntoskrnl/ke/amd64/stubs.c

```diff
@@ -330,6 +330,9 @@ KiSystemCallHandler(
     TrapFrame->TrapFrame = (ULONG64)Thread->TrapFrame;
     Thread->TrapFrame = TrapFrame;
 
+    /* We don't have an exception frame yet */
```

Comment:
Does "yet" here refer to an unimplemented feature, i.e. we should be doing something like the x86 version
```
1754     TrapFrame->ExceptionList = KeGetPcr()->NtTib.ExceptionList;
1755     KeGetPcr()->NtTib.ExceptionList = EXCEPTION_CHAIN_END;
```
? Or does it simply mean we'll do this at some other point in the code?

Reply:
You are confusing ExceptionList with Exception frame. The former only exists on x86 as all other architectures use table based SEH. The latter is to store non-volatile register information on an exception. On non-x86 this isn't done on a syscall to avoid the performance overhead of saving all the registers, when it's not needed (with one exception: NtSetContextThread, which needs to restore non-volatiles from the context when returning, but for that there is a workaround)